pkg/server/render.go

```diff
@@ -60,9 +61,26 @@ func (ctrl *Controller) renderHandler(w http.ResponseWriter, r *http.Request) {
 		out = &storage.GetOutput{Tree: tree.New()}
 	}
 
-	fs := out.Tree.FlamebearerStruct(p.maxNodes)
-	res := renderResponse(fs, out)
-	ctrl.writeResponseJSON(w, res)
+	switch p.format {
+	case "json":
+		fs := out.Tree.FlamebearerStruct(p.maxNodes)
+		res := renderResponse(fs, out)
+		ctrl.writeResponseJSON(w, res)
+	case "pprof":
+		pprof := out.Tree.PprofStruct(&tree.PprofMetadata{
+			Type: out.SpyName, // TODO
```

Comment:
@petethepig @abeaumont Don't think it's correct to use `SpyName` here. Is there a way to extract the type of the profile?